### PR TITLE
Doubleclick  floodlight tag tealium migration - Car UDP

### PR DESCRIPTION
**Pull request by j-ibrahim**
Doubleclick  floodlight tag tealium migration - Car UDP

### CODE REVIEW

**Comment by j-ibrahim** [comment]
image

image

**Comment by j-ibrahim** [comment]
image

**Comment by j-ibrahim** [comment]
image

File: src/library/simplified/extension/price-per-day.js [hunk 1]
```diff
@@ -12,4 +12,7 @@ if (utag.isCarCO()) {
         pricePerDay = b['entity.checkout.car.pricePerDay'];
     } 
     b['pricePerDay'] = pricePerDay;
+}
+else if (utag.isCarUDP() && typeof b["entity.carDetails.carCategoryNameLocalized"] !== undefined) {
+    b['pricePerDay'] = b["entity.carDetails.carCategoryNameLocalized"];
```
Comment (tyang-expedia) [inline src/library/simplified/extension/price-per-day.js]: Is this correct? Do we want the category name to go into the price per day?

Comment (tyang-expedia) [inline src/library/simplified/extension/price-per-day.js]: Is this correct? Do we want the category name to go into the price per day?

**Review by tyang-expedia — CHANGES_REQUESTED** [review]

File: src/library/simplified/extension/price-per-day.js (hunk 1 above)
Comment (j-ibrahim) [reply src/library/simplified/extension/price-per-day.js]: tony, thanks for the comment. I missed it while creating PR. Incorporated that and updated PR

**Review by tyang-expedia — APPROVED** [review]

**Pull request merged**